packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -68,23 +79,44 @@ function BasicColorList({
   allowsGradient,
 }) {
   const colorAsBackground = getPatternAsString(color);
+  const listRef = useRef(null);
+
+  useRovingTabIndex({ ref: listRef });
+
+  const selectedSwatchList = colors
+    .map((pattern, i) => {
+      const patternAsBackground = getPatternAsString(pattern);
+      const isSelected = colorAsBackground === patternAsBackground;
+      if (isSelected) {
+        return i;
+      }
+      return false;
+    })
+    .filter((c) => c !== false);
```

Comment:
Well, I guess there's only one that can be selected, though?

So `findIndex` is probably a reasonable version:

```suggestion
  const selectedSwatchIndex = colors
    .map(getPatternAsString)
    .findIndex((c) => colorAsBackground === c);
```